src/Explorer/Tables/TableEntityProcessor.ts

```diff
@@ -183,12 +183,14 @@ export function convertEntityToNewDocument(entity: Entities.ITableEntityForTable
           parsedValue = DateTimeUtilities.convertJSDateToTicksWithPadding(propertyValue);
           break;
         case Constants.TableType.Boolean:
-          parsedValue = propertyValue.toLowerCase() === "true";
+          parsedValue = propertyValue.toString().toLowerCase() === "true";
           break;
         case Constants.TableType.Int32:
-        case Constants.TableType.Int64:
           parsedValue = parseInt(propertyValue, 10);
           break;
+        case Constants.TableType.Int64:
+          parsedValue = parseInt(propertyValue, 10).toString();
```

Comment:
Remove `parseInt` here. JS will parse this as a double, but in reality it could be larger than that. Just return the property value directly.

Reply:
Done